pxf/pxf-hdfs/src/test/java/org/apache/hawq/pxf/plugins/hdfs/ChunkReaderTest.java

```diff
@@ -1,18 +1,15 @@
-package com.pivotal.pxf.plugins.hdfs;
+package org.apache.hawq.pxf.plugins.hdfs;
 
 import org.apache.hadoop.io.Writable;
 import org.apache.hadoop.hdfs.DFSInputStream;
 
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mockito;
 import org.mockito.stubbing.*;
 import org.mockito.invocation.*;
-import org.powermock.api.mockito.PowerMockito;
 import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.core.classloader.annotations.SuppressStaticInitializationFor;
```

Comment:
I assume you did change through IDE - any reason IDE would remove these name spaces with this refactory?